docker/make_docker_image.sh

```diff
@@ -18,5 +18,5 @@
 
 set -e
 
-#cd lightgbm-ci-build-env
+cd docker
```

Comment:
Isn't it `docker/lightgbm-ci-build-env`?

Reply:
This must execute from the folder above lightgbm-ci-build-env so it can build the docker image. Don't worry, I tested nuking everything and it worked :)

Reply:
Can we execute `docker build -t lightgbm-ci-build-env lightgbm-ci-build-env` (below line) without a dockerfile on the current path? :thinking:

Reply:
NVM I see that it uses the folder `lightgbm-ci-build-env`